crates/assists/src/handlers/generate_enum_match_method.rs

```diff
@@ -1,10 +1,11 @@
+use ast::Adt;
```

Comment:
https://github.com/rust-analyzer/rust-analyzer/blob/master/docs/dev/style.md#import-style

Reply:
ah, I wasn't aware -- thanks for pointing that out. Thanks!